FBSDKCoreKit/FBSDKCoreKit/FBSDKGraphErrorRecoveryProcessor.m

```diff
@@ -137,6 +137,7 @@ - (BOOL)processError:(NSError *)error request:(FBSDKGraphRequest *)request deleg
 
 #pragma mark - UIAlertViewDelegate
 
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Can you surround the statement with `#pragma clang diagnostic push` and `#pragma clang diagnostic pop` so that it doesn't suppress other code?